core/src/main/java/de/muenchen/allg/itd51/wollmux/OpenExt.java

```diff
@@ -228,11 +225,11 @@ public List<String> getPrograms()
   }
 
   /**
-   * Legt url als die Datei fest mit der die externe Anwendung gestartet werden soll.
-   * Ob die URL selbst als Parameter an das Programm übergeben wird oder ob der
-   * Inhalt der URL heruntergeladen und in eine Datei gespeichert wird, deren Pfad
-   * als Parameter übergeben wird, wird durch die DOWNLOAD-Angabe in der Definition
-   * der externen Anwendung in der wollmux.conf bestimmt.
+   * Specifies URL as the file with which the external application shall be started.
```

Comment:
Specifies url (not URL) - it refers to the parameter.
Maybe write `url` with upticks to be clearer.